src/variables/actions/thunks.ts

```diff
@@ -228,6 +229,8 @@ export const createVariable = (
       throw new Error(resp.data.message)
     }
 
+    event('variable.created')
```

Comment:
```suggestion
    event('variable.created.from_blank', {id: resp.data.id, name: variable.name})
```

let's add some more context here